client-v2/src/main/java/com/clickhouse/client/api/insert/InsertResponse.java

```diff
@@ -85,4 +85,12 @@ public long getServerTime() {
     public long getResultRows() {
         return operationMetrics.getMetric(ServerMetrics.RESULT_ROWS).getLong();
     }
+
+    /**
+     * Alias for {@link ServerMetrics#QUERY_ID}
+     * @return number of returned bytes
+     */
+    public String getQueryId() {
+        return operationMetrics.getMetric(ServerMetrics.QUERY_ID).getString();
```

Comment:
It is a good idea to put query ID close to metrics, because eventually it will be a metric label. 
Would you please add `labels` to operations metrics and put query id there?